spec/routing/bulletins_routing_spec.rb

```diff
@@ -31,7 +29,7 @@
     let(:post_bulletin) { post "/v1/bulletins" }
 
     it 'routes to bulletins#create)' do
-      expect(post_bulletin).to route_to(controller: 'basechurch/v1/bulletins',
+      expect(post_bulletin).to route_to(controller: 'v1/bulletins',
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.